jekyll/_cci2_ja/create-project.adoc

```diff
@@ -5,6 +5,7 @@ contentTags:
   - クラウド
   - Server v4.x
   - Server v3.x
+  - Server v2.x
```

Comment:
@yuki-sakamoto-san I see all the instances where I manually removed server 2 from the page tags crowdin wants to put them back. Could you remove them via Crowdin? Or if I can get access I could do stuff like this :-D